### PR TITLE
Add roadmap 2022

### PR DESCRIPTION
**Pull request by curquiza**
⚠️ This is the core-team roadmap and NOT a product roadmap. This roadmap completes the roadmap of the product team but is not a duplication!

Sorry for the delay. The content should be not a surprise, only topics we discussed I finally put in public 😇

### CODE REVIEW

**Review by qdequele — COMMENTED** [review]
I think it's really good. I just think you don't have to go into the features that are going to be coded in the `Release the game-changer features 🛠` part. Apart from that it's really good!

File: roadmaps/2022.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+We strongly believe building a product in open source is the safest way to provide a product that will last in the long term. We indeed think the community is one of our biggest advantages since we get contributions, but also feedback that is also very valuable. Plus, having the code in public enforce us to work in a clean and clear way. This makes it more reliable in the long term.
+
+We want to increase the participation regarding fixes (quick or not), technical discussions where we would need more insights, and any feedback in general.
+
+### Recruitments 🦀
+
+Since we want to focus on more technical tasks, the core team needs to hire experts and senior people.
+
+We plan to start an internship program too. The goal is to make Meilisearch more visible but also to attract young developers by making them discover open-source and fall in love with Rust.
```
[the remark this answers is not in the document]
Comment (curquiza) [reply roadmaps/2022.md:29]: You're right, I'm nuancing this now

**Comment by curquiza** [comment]
> I think it's really good. I just think you don't have to go into the features that are going to be coded in the Release the game-changer features 🛠 part.

If you agree I would let the list. It's not the detailed feature but the main implementations. Indexation speed is not only a feature but mostly R&D, same for the tokenizer improvement. I think this list is a good way to remind on what the core team worked concretely during the year, without going into details

The title is maybe not appropriate saying this! I'm changing this!

File: roadmaps/2022.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+### Improve the search engine experience 🛠
+
+Regarding this point, more details are available on the product team roadmap, but here are the main goals:
+- Tenant tokens
+- Nested fields
+- Federated search
+- Enhance some customization
+- Improve the indexation speed
+- Improve the tokenizer
+- Stabalize the API routes, essential for the Cloud team
```
Comment (Kerollmops) [inline roadmaps/2022.md:58]:
```suggestion
- Stabilize the API routes, essential for the Cloud team
```

**Review by Kerollmops — APPROVED** [review]
Looks good to me, nothing more to add. Thank you Curqui!

**Review by gmourier — APPROVED** [review]

**Review by Kerollmops — APPROVED** [review]

**Review by qdequele — APPROVED** [review]

**Pull request merged**